refactron/cicd/pre_commit.py

```
@@ -0,0 +1,162 @@
+    critical = summary.get("critical", 0)
+    errors = summary.get("errors", 0)
+    warnings = summary.get("warnings", 0)
+    
+    print(f"📊 Refactron Pre-Commit Analysis:")
+    print(f"  Critical: {{critical}}")
+    print(f"  Errors: {{errors}}")
+    print(f"  Warnings: {{warnings}}")
+    
+    fail = False
+    
+    if {str(fail_on_critical)} and critical > {max_critical}:
```

Ruff (0.14.13)

Use explicit conversion flag

Replace with conversion flag

(RUF010)